Resources/Prototypes/_Sunrise/Surgery/attachment_surgeries.yml

```
@@ -0,0 +1,223 @@
+- type: entity
+  parent: SurgeryBase
+  id: SurgeryLimbAttachmentHead 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)